src/main/java/Duke.java

```diff
@@ -28,11 +28,11 @@ private static void load() throws IOException, DukeException {
             switch(type) {
                 case "E":
                     String at = commandArr[3];
-                    tasks.add(new Event(description, status, at));
+                    tasks.add(new Event(description, at, status));
                     break;
                 case "D":
```

Comment:
I believe the coding standard states no indentation for case clauses in switch cases. Should not affect the code though!